client/src/components/Login/index.js

```diff
@@ -8,113 +8,126 @@ import { LOGIN_USER } from '../../utils/mutations';
 import Auth from '../../utils/auth';
 
 function Login() {
-	const [isSignOpen, setIsSignOpen] = useState(false);
+  const [isSignOpen, setIsSignOpen] = useState(false);
+  let navigate = useNavigate();
+  const toggleSign = () => {
+    setIsSignOpen(!isSignOpen);
+  };
 
-	const toggleSign = () => {
-		setIsSignOpen(!isSignOpen);
-	};
+  const [formState, setFormState] = useState({ email: '', password: '' });
+  const [login, { error }] = useMutation(LOGIN_USER);
 
-	const [formState, setFormState] = useState({ email: '', password: '' });
-	const [login, { error }] = useMutation(LOGIN_USER);
+  // update state based on form input changes
+  const handleChange = (event) => {
+    const { name, value } = event.target;
 
-	// update state based on form input changes
-	const handleChange = (event) => {
-		const { name, value } = event.target;
+    setFormState({
+      ...formState,
+      [name]: value,
+    });
+  };
 
-		setFormState({
-			...formState,
-			[name]: value,
-		});
-	};
+  // submit form
+  const handleFormSubmit = async (event) => {
+    event.preventDefault();
+    console.log('handleFormSubmit login');
+    try {
+      console.log('==============logging in===================');
+      console.log(formState);
+      const { data } = await login({
+        variables: { ...formState },
+      });
+      console.log('====================login==========================');
+      console.log(data);
+      setTimeout(() => {
+        Auth.login(data.login.token);
+      }, 10000);
+      setTimeout(() => {
+        navigate('/dashboard');
+      }, 10000);
+    } catch (e) {
+      console.log('============ERRRRRRORRRRR====================');
+      console.error(e);
+    }
 
-	// submit form
-	const handleFormSubmit = async (event) => {
-		event.preventDefault();
+    // clear form values
+    setFormState({
+      email: '',
+      password: '',
+    });
+  };
 
-		try {
-			const { data } = await login({
-				variables: { ...formState },
-			});
+  return (
+    <>
+      <div className={`dropdown is-right ${isSignOpen ? 'is-active' : ''}`}>
+        <div className="dropdown-trigger">
```

Comment:
Double quotes.

Reply:
line 62 is a backtick
line 63 is intended as double quotes because is a JSX.